### ISSUE TITLE
About error “more than one element of the written-to tensor refers to a single memory location.”

### ISSUE THREAD

**Issue by danelee2601** [huangleiBuaa/IterNorm-pytorch#2]
If I use `./extension/normalization/iterative_normalization.py` directly, I get the error - "more than one element of the written-to tensor refers to a single memory location". This error occurs due to the PyTorch version update. I could find the solution [here](https://discuss.pytorch.org/t/about-error-more-than-one-element-of-the-written-to-tensor-refers-to-a-single-memory-location/85526).<br>
Basically,
change `self.register_buffer('running_wm', torch.eye(num_channels).expand(num_groups, num_channels, num_channels)` to `self.register_buffer('running_wm', torch.eye(num_channels).expand(num_groups, num_channels, num_channels).clone())`.

**Comment by huangleiBuaa** [comment]
@danelee2601 ， thanks for your constructive comments.
